libcxx/test/std/input.output/file.streams/fstreams/filebuf/types.pass.cpp

```diff
@@ -24,6 +24,10 @@
 
 #include "test_macros.h"
 
+#if _LIBCPP_STD_VER >= 26
```

Comment:
This is the way to test for versions in the test suite.
```suggestion
#if TEST_STD_VER >= 26
```
The same for other places. Also since we can no longer use `native_handle_test_helpers` I prefer the use the C++ version guard in that header. (I know I asked for it's removal earlier, sorry).